Plugin~/unity/include/IUnityGraphics.h

```diff
@@ -34,6 +34,12 @@ typedef enum UnityGfxDeviceEventType
     kUnityGfxDeviceEventAfterReset     = 3,
 } UnityGfxDeviceEventType;
 
+typedef enum UnityEncoderType
```

Comment:
This class has moved from the other repo. It is better not to touch it.
https://github.com/Unity-Technologies/NativeRenderingPlugin/tree/master/PluginSource/source/Unity

It is better to put this in `pch.h`.